src/Umbraco.Headless.Client.Net/Constants.cs

```diff
@@ -23,7 +23,7 @@ public static class Constants
         public const string ApiMinimumVersion = "2.0";
         public const string ApiMinimumVersionHeader = Headers.ApiVersion + ": " + ApiMinimumVersion;
         //Current version for this library
-        public const string ApiVersion = "2.2";
+        public const string ApiVersion = "2.3";
```

Comment:
Before we change this I think we need to be sure that nothing is using/expecting something like 2.1 or 2.2 as the version header. It looks like both Content and Media uses MinimumApiVersion, but are there any endpoints that explicitly changes this to something else?

Reply:
Im not sure I follow - I think the endpoints we define to be called by the client has the headers explicitly set, and currently none of them uses this string as far as i'm aware

Reply:
Actually I'm not really sure what ``APIVersionHeader`` is being used for? nothing in the client library references it, only ApiMinimumVersionHeader is being referenced

Reply:
Okay, well if nothing is using it then there is no cow on the is 😉